csrankings.csv

```diff
@@ -2142,6 +2144,7 @@ Benjamin Rossman,University of Toronto,http://www.math.toronto.edu/rossman,pWJ0S
 Benjamin Ruddell,Northern Arizona University,https://nau.edu/siccs/faculty/benjamin-ruddell,W1fB5OcAAAAJ
 Benjamin Stock,CISPA Helmholtz Center,https://people.cispa.io/ben.stock,_hDa-5wAAAAJ
 Benjamin Swift,Australian National University,https://cs.anu.edu.au/people/ben-swift,OQdYgLEAAAAJ
+Benjamin Ujcich,Georgetown University,https://personal.benujcich.georgetown.domains/,cb0qEdoAAAAJ

```

Comment:
You need to add an entry to `dblp-aliases.csv` for this faculty member.

Reply:
Done